MovieApp/Screens/Home/Views/HomeMovieCell/HomeMovieCell.swift

```diff
@@ -18,21 +19,17 @@ class HomeMovieCell: UITableViewCell {
 	@IBOutlet private weak var playButton: UIButton!
 	@IBOutlet private weak var infoButton: UIButton!
 	@IBOutlet private weak var addMyListButton: UIButton!
-	
-	// MARK: - Variables
-	private var viewModel: IMovieViewModel!
-	// MARK: - Life cycles
+	@IBOutlet private weak var imageHomeMovieCell: UIImageView!
+
+// MARK: - Life cycles
 	override func awakeFromNib() {
 		super.awakeFromNib()
-		self.backgroundView = UIImageView(image: UIImage(named: "image2"))
 	}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Code4Fun-Group/Movie-App/blob/6d8da324e91a4d6141b0bbfcfe90b541f3760d2d/MovieApp/Screens/Home/Views/HomeMovieCell/HomeMovieCell.swift#L27"></span>Override methods which only call super can be removed</td>
    </tr>
  </tbody>
</table>